content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+  prevents Kubernetes from automatically creating these tokens for
+  ServiceAccounts.
+
+## Authenticating service account credentials {#authenticating-credentials}
+
+ServiceAccounts use signed
+{{<glossary_tooltip term_id="jwt" text="JSON Web Tokens">}}  (JWTs)
+to authenticate to the Kubernetes API server, and to any other system where a
+trust relationship exists. Depending on the token issuer, a ServiceAccount
+token might also have an expiry time, an audience, and a time after which the
+token *starts* being valid. When a process running in a Pod attempts to
+communicate with the Kubernetes API server, it adds an
+`Authorization: Bearer <token>` header to the HTTP request. The API server
+checks the validity of the bearer token as follows:
+
+1. Check the token signature.
+1. Check whether the token has expired.
+1. Check whether the token is currently valid.
+1. Check the audience claims.
+
+For tokens issued using the `TokenRequest` API, the API server also checks that
+the specific Pod using the ServiceAccount still exists, matching by the
+{{< glossary_tooltip term_id="uid" text="unique ID" >}} of that Pod. For legacy
+tokens that are mounted as Secrets in Pods, the API server checks the token
+against the Secret.
+
+For more information about the authentication process, refer to
+[Authentication](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+### Authenticating service account credentials in your own code {authenticating-in-code}
+
+If you have services of your own that need to validate Kubernetes service
+account credentials, you can use the following methods:
+
+* [TokenReview API](/docs/reference/kubernetes-api/authentication-resources/token-review-v1/)
+  (recommended)
+* OIDC discovery
+
+The Kubernetes project recommends that you use the TokenReview API, because this
+method invalidates tokens that are bound to API objects such as Secrets,
+ServiceAccounts, and Pods when those objects are deleted. For example, if you
+delete the Pod that contains a projected ServiceAccount token, the TokenReview
+API invalidates that token immediately. If you use OIDC validation instead, your
+clients continue to treat the token as valid until the token reaches its
+expiration timestamp.
+
+## Alternatives
+
+* Issue your own tokens using another mechanism, and then use
+  [Webhook Token Authentication](/docs/reference/access-authn-authz/authentication/#webhook-token-authentication)
+  to validate bearer tokens using your own validation service.
+* Provide your own identities to Pods.
+    * [Use the SPIFFE CSI driver plugin to provide SPIFFE SVIDs as X.509 certificate pairs to Pods](https://cert-manager.io/docs/projects/csi-driver-spiffe/).
+    * [Use a service mesh such as Istio to provide certificates to Pods](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/).
```

Comment:
is it ok to mention 3rd parties here?

Reply:
Yes, it is. We should use a thirty party content shortcode to mark that we're doing so.

Reply:
Done